examples/nextjs-with-typescript/pages/MuxVideo.tsx

```diff
@@ -45,7 +45,7 @@ function MuxVideoPage() {
           <input
             id="paused-control"
             type="checkbox"
-            onChange={() => setPaused(!paused)}
+            onChange={() => paused ? mediaElRef.current.play() : mediaElRef.current.pause()}
```

Comment:
suggestion(non-blocking): consider keeping this as is and instead doing the `play()`/`pause()` in a `useEffect(() => {}, [paused]);`